.travis.mk

```diff
@@ -317,13 +315,13 @@ deps_osx:
 	# try to install the packages either upgrade it to avoid of fails
 	# if the package already exists with the previous version
 	brew install --force ${OSX_PKGS} || brew upgrade ${OSX_PKGS}
-	pip install --force-reinstall -r test-run/requirements.txt
+	pip3 install --user --force-reinstall -r test-run/requirements.txt
```

Comment:
I don't see any usage of this code. It seems, it was used on GitLab CI, but now it does not.

Reply:
`--user` was removed, pip -> pip3 is kept. Anyway, okay.